corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -77,6 +79,46 @@ def test_case_search_action(self):
             "./detail[@id='m0_search_short']/action"
         )
 
+    @flag_enabled('SPLIT_SCREEN_CASE_SEARCH')
+    def test_splitscreen_case_search_removes_search_again(self):
+        app = Application.wrap(self.get_json('suite-advanced'))
+        app.modules[0].search_config = CaseSearch(
+            search_label=CaseSearchLabel(
+                label={'en': 'Search All Cases'},
+            ),
+            search_again_label=CaseSearchAgainLabel(
+                label={'en': 'Search Again'},
+            ),
+            properties=[CaseSearchProperty(name='name', label={'en': 'Name'})],
+        )
+        form_id = app.modules[0].forms[0].unique_id
+        app.modules[0].case_list_form = CaseListForm(
+            form_id=form_id,
+            label={'en': 'Registration Form'},
+        )
+        app = Application.wrap(app.to_json())
```

Comment:
Why is this line needed?

Reply:
I used the setup structure from the [test above it](https://github.com/dimagi/commcare-hq/blob/5f7be3abcde31b37285e1ca0c08f7557a3780593/corehq/apps/app_manager/tests/test_suite.py#L38-L53) but didn't include the comment: `# wrap to have assign_references called`

Reply:
I'd vote for just calling `app.modules[0].assign_references()` then, but this isn't a blocking comment.